src/views/UserEdit/UserEdit.js

```diff
@@ -292,6 +292,11 @@ class UserEdit extends React.Component {
     }
 
     // data is information that the form needs, mostly to populate options lists
+
+    // check to see if we are editing an existing user, and if the data on that user has finished loading.
+    if (!this.getUser() && this.props.match.params.id) {
+      return null;
+    }
```

Comment:
Would it make more sense, or perhaps be a better UX, to incorporate this check into the `resourcesLoaded` check above and return a `<LoadingView>` instead of `null`?

Reply:
I tried it out-it makes that "if" a bit long, but it does present a loading indicator while it's loading the data.  Pushed the change up just now.

Reply:
`params.id` is already destructured, which would help with the line-length a bit. 

You could leave it as-is, or put the separate condition-groups on separate lines if it bugs you.